tests/LabelAssertionTests.py

```diff
@@ -0,0 +1,63 @@
+import unittest
+import json
+from PullRequest import PullRequest
+
+NO_LABELS = ['']
+MOBILE_PRODUCT_LABELS = ['product/all-users', 'product/custom', 'product/invisible', 'product/internal-or-flagged']
+MULTIPLE_LABELS = ['cross requested', 'product/internal-or-flagged', 'ready for review']
+
+# labels on this PR are: product/invisible
+PR_1857_URL = 'https://api.github.com/repos/dimagi/commcare-android/issues/1857'
```

Comment:
I think these tests make real requests to the github api endpoint. It might be better to add sample json files to test the minimal functionality you are testing. If we put this on to a CI tool, this will probably get rate limited.

Reply:
Good call, fixing this now